src/components/Patient/SampleTestCard.tsx

```diff
@@ -105,7 +104,7 @@ export const SampleTestCard = (props: SampleDetailsProps) => {
               id="sample-test-status"
               className="mt-1 overflow-x-scroll whitespace-normal break-words text-sm font-medium leading-5"
             >
-              {startCase(camelCase(itemData.status))}
+              {startCase(camelCase(String(itemData.status)))}
```

Comment:
`capitalize` class

Reply:
ok sir I will add capitalize
but actually earlier my cypress tests were failing , so i did this
I know the working is same
"swanand bhuskute" becomes "Swanand Bhuskute"
but "swanandbhuskute" just becomes "Swanandbhuskute"

i will try and revert

Reply:
isn't that expected?